Makefile

```diff
@@ -312,6 +312,31 @@ ca-geo:
 		--module tasks.ca.statcan.geo AllGeographies \
 		--parallel-scheduling --workers=8
 
+### es
+es-all: es-cnig es-ine es-lacaixa
+
+es-cnig:
+	docker-compose run --rm bigmetadata luigi \
+		--module tasks.es.cnig AllGeometries \
+		--parallel-scheduling --workers=8
+
+es-ine: es-ine-phh es-ine-fyp
+
+es-ine-phh:
+	docker-compose run --rm bigmetadata luigi \
+		--module tasks.es.ine PopulationHouseholdsHousingMeta \
+		--parallel-scheduling --workers=8
+
+es-ine-fyp:
+	docker-compose run --rm bigmetadata luigi \
+		--module tasks.es.ine FiveYearPopulationMeta \
+		--parallel-scheduling --workers=8
+
+es-lacaixa:
```

Comment:
There are some parameters missing for this one. We have loaded Anuarios for 2008, 2011 and 2013. I think we need to run this task multiple times, one per timespan. Hard to test though, because it is broken 😆

Reply:
Yes, I'm planning to solve this in #259